src/components/LocationSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Location } from '../types/sparkeats';
+
+type LocationSelectProps = {
+  locations: Location[];
+};
+
+function LocationSelect({ locations }: LocationSelectProps) {
+  const allCities = locations.map((location) => location.city);
+  const uniqueCities = [...new Set(allCities)];
+
+  return (
+    <div className="location-select">
+      <h2 className="location-select__title">Pick a Location</h2>
+
+      <select className="location-select__menu" name="location" id="location">
```

Comment:
We will also need state for the selected city and an `onChange` event handler to update the selected city and filter the location cards, however, this can be done in #432.